src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -452,16 +455,18 @@ def set_advanced_networking(
         else:
             machine_networks = [models.MachineNetwork(cidr=cidr) for cidr in machine_networks]
 
+        vip_dhcp_allocation_var = {}
```

Comment:
```suggestion
        extra_vars = {}
```
let's make this generic from start, I have a feeling we'll have a reason to append more to it in the future

Reply:
More generic than `extra_vars`?